### PR TITLE
fix(settings): test account default hostname filter

### LINKED ISSUE #8564: Bug: Filter not showing fully
## Bug description

Think this is a bug. 

Internal users filter config. Click one existing filter with a single value, get this, with the values editable:

Screenshot 2022-02-11 at 14 22 33

Click one existing filter with multiple comma separated values, get this, without editable values:

Screenshot 2022-02-11 at 14 22 48

This was on Hogflix. 

## Expected behavior

Editable values. 

## How to reproduce

1. See above.
2. Read what's above. 
3. Reproduced. 

## Environment

- [ ] PostHog Cloud
- [ ] self-hosted PostHog (ClickHouse-based), version/commit: _please provide_
- [ ] self-hosted PostHog (Postgres-based, legacy), version/commit: _please provide_

### PR DESCRIPTION
**Pull request by mariusandra**
## Problem

Fixes https://github.com/PostHog/posthog/issues/8564

## Changes

We didn't set the filter type (`"event"`) for the default test account filters. This fixes that, and adds a patch to default filters to "event" if editing, and a value is set, but no type.

### Before
image

### After
image

### CODE REVIEW

File: frontend/src/lib/components/PropertyFilters/utils.ts [hunk 1]
```diff
@@ -2,6 +2,16 @@ import { PropertyGroupFilter, AnyPropertyFilter, EventDefinition, PropertyFilter
 import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
 import { flattenPropertyGroup, isPropertyGroup } from 'lib/utils'
 
+export function sanitizePropertyFilter(propertyFilter: AnyPropertyFilter): AnyPropertyFilter {
```
Comment (Twixes) [inline frontend/src/lib/components/PropertyFilters/utils.ts:5]: Maybe add a comment explaining why this function is a thing

File: frontend/src/lib/components/PropertyFilters/utils.ts [hunk 2]
```diff
@@ -2,6 +2,16 @@ import { PropertyGroupFilter, AnyPropertyFilter, EventDefinition, PropertyFilter
...
+    if (!propertyFilter.type && propertyFilter.value) {
```
Comment (Twixes) [inline frontend/src/lib/components/PropertyFilters/utils.ts:6]: Why check for `propertyFilter.value` too? Note: "is set"/"is not set" doesn't have to have a `value`.

**Comment by Twixes** [comment]
Bump?

**Review by mariusandra — COMMENTED** [review]
Is this more sleek? @Twixes

**Review by Twixes — APPROVED** [review]
[As slick as a buttered eel on a marble floor.](https://similesmiles.com/slick)

**Pull request merged**